### PR TITLE
Send to grave

### PR DESCRIPTION
**Pull request by Ledespana**

### CODE REVIEW

File: frontend/components/SendToGravePanel.tsx [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+import LSP8IdentifiableDigitalAsset from '@lukso/lsp-smart-contracts/artifacts/LSP8IdentifiableDigitalAsset.json';
+import { LSP4_TOKEN_TYPES } from '@lukso/lsp-smart-contracts';
+import {
+  TokenData,
+  getEnoughDecimals,
+  getLSPAssetBasicInfo,
+} from '@/utils/tokenUtils';
+import { LSP4TokenTypeValues } from '@lukso/lsp-factory.js/build/main/src/lib/interfaces/digital-asset-deployment';
+
+export default function ManageAllowList() {
```
Comment (democrazyfi) [inline frontend/components/SendToGravePanel.tsx:25]: Minor component name needs updating

Comment (Ledespana) [reply frontend/components/SendToGravePanel.tsx:25]: good catch

**Review by tehnalogos — APPROVED** [review]
looks good

File: frontend/components/SendToGravePanel.tsx [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+    );
+    setCanSubmit(true);
+  };
+
+  const removeTokenFromAllowList = async () => {
+    const LSP1GraveForwarderContract = new ethers.Contract(
+      networkConfig.universalGraveForwarder,
+      LSP1GraveForwarderAbi.abi,
+      signer
+    );
```
Comment (democrazyfi) [inline frontend/components/SendToGravePanel.tsx:129]: minor but would be good to use the typechain version of this to get type saftey / checks

```
      const LSP1GraveForwarderContract = LSP1GraveForwarder__factory.connect(
        networkConfig.universalGraveForwarder,
        signer
      );
```

File: frontend/components/SendToGravePanel.tsx [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+              }}
+            />
+            <Text ml={2} fontFamily="Bungee" fontWeight={400} fontSize={'14px'}>
+              {FieldMessage()}
+            </Text>
+          </Flex>
+        </Box>
+      </FormControl>
+      <Flex justifyContent={'flex-start'} gap={2} width={'100%'}>
+        <Button
```
Comment (democrazyfi) [inline frontend/components/SendToGravePanel.tsx:324]: Should this have a loadingText={FieldMessage()}? 

That way we remove the text field above and our button will automatically change text based on the status, as that's what we do with other buttons like revive

File: frontend/components/SendToGravePanel.tsx (hunk 2 above)
Comment (tehnalogos) [reply frontend/components/SendToGravePanel.tsx:128]: done

File: frontend/components/SendToGravePanel.tsx (hunk 3 above)
Comment (tehnalogos) [reply frontend/components/SendToGravePanel.tsx:327]: @democrazyfi those messages are much more general than just loading states. although in general I agree we should standardize this, we can do that during a larger codebase refactor down the line

**Review by tehnalogos — APPROVED** [review]

**Pull request merged**